### ISSUE TITLE
Is QuickSync `video decoding not supported anymore?

### ISSUE THREAD

**Issue by psisis** [Nevcairiel/LAVFilters#327]
I've been having troubles activating QuickSync hardware acceleration for Lav Video decoder. When I googled for issues I found this [issue](https://github.com/nevcairiel/lavfilters/issues/79) from 2016 where the (dev?) said the problems would be in the system and not LAV software itself. But in my case at least that is not true. Potplayer and FFMpeg uses Quicksync just fine. It's worth using it, because it takes the load off Madvr rendering with up to 5ms with is kinda substantial.

**Comment by clsid2** [comment]
DXVA2 and D3D11 use the exact same hardware decoder on your Intel GPU. You should use of the those instead of QuickSync. The QuickSync implementation in LAV is obsolete, outdated, and has no benefits over the other hardware acceleration choices. It will most likely be removed in the future.

**Comment by Nevcairiel** [comment]
As clsid2 points out, QuickSync and DXVA/D3D11 all use the same hardware, so those are recommended instead.

**Comment by 2blackbar** [comment]
> DXVA2 and D3D11 use the exact same hardware decoder on your Intel GPU. You should use of the those instead of QuickSync. The QuickSync implementation in LAV is obsolete, outdated, and has no benefits over the other hardware acceleration choices. It will most likely be removed in the future.

You are wrong dood, i have tried quicksync vs ffmpeg and dxva accelerators, quicksync gives me 2%cpu load and other ones give 9, sure maybe theyre integrated now but integration is not that great, better to use separate quicksync to decode instead of the one thats in dxva
